File: ulohy/31-transpozice/reseni/main.cs
using System;

internal class Program
{
    static void Main(string[] args)
    {
        int w = int.Parse(Console.ReadLine());
        int h = int.Parse(Console.ReadLine());
        Console.ReadLine();

        if (w < 1 || h < 1)
        {
            return;
        }

        char[,] table = new char[h, w];

        for (int row = 0; row < h; row++)
        {
            for (int col = 0; col < w; col++)
            {
                table[row, col] = char.Parse(Console.ReadLine());
            }
        }

        for (int col = 0; col < w; col++)
        {
            for (int row = 0; row < h; row++)
            {
                Console.Write(table[row, col]);
            }

            Console.WriteLine();
        }
    }
}

File: ulohy/31-transpozice/reseni/main.py
import sys

if __name__ == "__main__":
    w = int(input())
    h = int(input())
    input()

    if w < 1 or h < 1:
        sys.exit()

    table = [["" for _ in range(w)] for _ in range(h)]

    for row in range(h):
        for col in range(w):
            table[row][col] = input()

    for col in range(w):
        for row in range(h):
            print(table[row][col], end="")

        print()
